examples/virtual-machines/linux/basic-password/main.tf

```
@@ -37,9 +37,9 @@ resource "azurerm_linux_virtual_machine" "main" {
   name                            = "${var.prefix}-vm"
   resource_group_name             = azurerm_resource_group.main.name
   location                        = azurerm_resource_group.main.location
-  size                            = "Standard_F2"
-  admin_username                  = "adminuser"
-  admin_password                  = "P@ssw0rd1234!"
+  size                            = "Standard_D2s_v3"
+  admin_username                  = "${var.username}"
+  admin_password                  = "${var.password}"
```

As these are not in the var file can we rever this?
```suggestion
  admin_username                  = "adminuser"
  admin_password                  = "P@ssw0rd1234!"
```